jerry-port/default/defaultx-handler.c

```diff
@@ -25,4 +33,14 @@ void
 jerryx_port_handler_print_char (char c) /**< the character to print */
 {
   printf ("%c", c);
+
+#ifdef JERRY_DEBUGGER
```

Comment:
I have concerns of using defines in the default port implementation. The documentation does not say anything about the debugger related part. IoT.js uses it's own port implementation in certain platforms, so these changes should be applied there as well, but documentation does not contain information about the debugger output.

Reply:
Note 1: Please note that the default port already contains debugger-specific code and macro guards (see `default-io.c`).

Note 2: Please also note my comment above. AFAIK, IoT.js doesn't use the external print handler from `jerry-ext/handler/handler-print.c` at all, which means that nothing in this file (debugger-specific or not) is used by IoT.js or its platform adaptations. Or am I missing something?

Reply:
1) Thanks for pointing it out. I did not know that.
2) My bad. You are right. It does not use the print handler of jerry-ext.